server/controllers/CommentController.js

```diff
@@ -1,25 +1,30 @@
+import uuidv4 from 'uuid/v4';
 import db from '../models';
 import response from '../helpers/response';
+import TokenManager from '../helpers/TokenManager';
 
+const id = uuidv4();
 
 const { Article, Comment } = db;
 
 /**
  * @class UserController
  */
 class CommentController {
-/**
- *
- * @description Allow user to comment on an article
- * @static
- * @param {*} req Express Request object
- * @param {*} res Express Response object
- * @returns {object} Json response
- * @memberof CommentController
- */
+  /**
+   *
+   * @description Allow user to comment on an article
+   * @static
+   * @param {*} req Express Request object
+   * @param {*} res Express Response object
+   * @returns {object} Json response
+   * @memberof CommentController
+   */
   static async createComment(req, res) {
     try {
-      const { userId, articleId, content } = req.body;
+      const token = req.headers.authorization.split(' ')[1];
```

Comment:
Why do you split Authorization? Are we using Bearer or a prefix before our token?

Reply:
Yes bearer comes with the token

Reply:
Our tokens are not returning Bearer